index.html

```diff
@@ -69,13 +69,13 @@ <h1 class="page-header"><a href="https://ptpb.pw/" target="_blank">ptpb.pw</a></
           <div class="col-xs-6 col-sm-6 form-inline form-group text-right">
             <div class="form-group">
               <div class="btn-group btn-block" data-toggle="buttons">
-                <label class="btn btn-default btn-block">
+                <label class="form-control">
```

Comment:
such layout breakage.

native checkboxes with labels are the hardest thing to do right.